### PR TITLE
prisma and planetscale

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: docs/guides/serverless-api-with-planetscale-and-prisma.md [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+  - [GCP](https://cloud.google.com)
+  - [Azure](https://azure.microsoft.com)
+
+To keep things light, the API we'll create generates memes by allowing uploads of image templates, then generating new images with overlaid text. Feel free to adapt the steps for any other API you'd like to build.
+
+> For image editing, we used a library from NPM called [jimp](https://www.npmjs.com/package/jimp), but you could use anything else you like.
+
+When you're done, you'll have an API which accepts a template image (like the left image below) and can generate multiple memes from it (like the right image below):
+
+<!-- TODO: include image ![meme example]() -->
```
Comment (davemooreuws) [inline docs/guides/serverless-api-with-planetscale-and-prisma.md:23]: Should we remove the image ref for now? Or did you want to add one?

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Looks good, just one thing.

**Review by tjholm — APPROVED** [review]
LGTM apart from Dave's comment.

File: docs/guides/serverless-api-with-planetscale-and-prisma.md (hunk 1 above)
Comment (jyecusch) [reply docs/guides/serverless-api-with-planetscale-and-prisma.md]: Thanks, I removed this one and fixed another image while I was at it

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**